index.html

```diff
@@ -0,0 +1,141 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <div class="logo">My Logo</div>
+      <div class="hum-menu">
+        <svg
+          width="40"
+          height="40"
+          viewBox="0 0 40 40"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+        >
+          <path
+            fill-rule="evenodd"
+            clip-rule="evenodd"
+            d="M11 15C11 14.4477 11.4477 14 12 14H28C28.5523 14 29 14.4477 29 15C29 15.5523 28.5523 16 28 16H12C11.4477 16 11 15.5523 11 15ZM11 20C11 19.4477 11.4477 19 12 19L28 19C28.5523 19 29 19.4477 29 20C29 20.5523 28.5523 21 28 21L12 21C11.4477 21 11 20.5523 11 20ZM12 24C11.4477 24 11 24.4477 11 25C11 25.5523 11.4477 26 12 26H28C28.5523 26 29 25.5523 29 25C29 24.4477 28.5523 24 28 24H12Z"
+            fill="#6070FF"
+          />
+        </svg>
+      </div>
+    </header>
+
+    <section class="headline">
+      <div class="headline-detail">
+        <div>
+          <h2 class="primary-text">I'm Mariane Glad to see you!</h2>
```

Comment:
- [ ] Since this is the main title of your portfolio, it would be better to use the `<h1>` tag instead of `<h2>`. 📜  This will improve the SEO of your page as result. You can read more about this topic [here](https://www.reliablesoft.net/h1-tag/). Kindly change it 🛠️

Reply:
Ok, thanks @anagudelogu